elements/rh-tooltip/rh-tooltip.ts

```diff
@@ -113,6 +114,10 @@ export class RhTooltip extends LitElement {
   override render() {
     const { alignment, anchor, open, styles } = this.#float;
 
+    const scheme = this.#style?.colorScheme ?? 'light';
```

Comment:
Oh ok, well then. If this is this easy, then I need to revisit primary-nav border-width issue. I think adding this simple js is an easy extra lines of code trade off.  Think we should we abstract this and implement as part of the @themeable decorator?